src/main/java/ch/njol/skript/expressions/ExprWardenAngryAt.java

```diff
@@ -0,0 +1,66 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Warden;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Warden Angry At")
+@Description({
+	"The entity a warden is most angry at.",
+	"A warden can be angry towards multiple entities with different anger levels.",
+})
+@Examples({
+	"if the most angered entity of last spawned warden is not last spawned pig:",
+		"\tset the most angered entity of last spawned warden to last spawned pig"
```

Comment:
Correct me if I'm wrong, but regardless of warden/pig... last spawned entity will only ever return the last spawned entity, therefore one of these would return null.

Reply:
Ahhh you're right, idk where  I got that impression from.